### ISSUE TITLE
[Feature Request] New slot if one Drawer is full

### ISSUE THREAD

**Issue by cresse2200** [jaquadro/StorageDrawers#321]
I have some hoppers feeding into the system, and if one drawer is full, it would be nice, if it just starts filling another unlocked drawer. So u can easier notice that the system is full, without running around and looking in all the drawers (For example look all but 4 and when something is filled in there u put it somewhere else).

**Comment by jaquadro** [comment]
It should already start filling another slot if the first one is full, as long as the slot is unlocked.  Not 100% sure on that behavior if you're going directly into a drawer, but it should definitely be true through a controller.

Is it just hoppers or other pipe systems too?

**Comment by cresse2200** [comment]
Ok, if it is a bug then first to my setup:
Minecraft 1.7.10
Forge 10.13.4.1614
Using the modpack "Life in the Woods"
Storage Drawers was 1.9.4 but Update to 1.10.0 made no difference

System.getProperty('os.arch') == 'amd64'
System.getProperty('java.version') == '1.8.0_101'
System.getProperty('java.vendor') == 'Oracle Corporation'
System.getProperty('sun.arch.data.model') == '64'

Other blocks interacting with the storage are minecraft hoppers and from the hopper ducts mod (hopper ducts and grated hoppers)

Tried feeding into the system with minecraft hoppers and hopper ducts, neither worked. I'm feeding into a slave controller. I locked my whole system with the key and then unlocked 6 2x2 drawers, where i want the excess to go. If i put something new in the system (with the hopper), there are no problems. They get placed in those 6 drawers. It just isn't accepting Items which have already a filled drawer.

I changed a couple of settings: 
1. withdraw a full stack with only one click
2. I doubled all the storage capabilities
3. I doubled the increase in storage capabilities
